language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/constants/CommandConstants.java

```diff
@@ -177,6 +178,12 @@ public class CommandConstants {
     public static final String EXTRACT_TO_CONSTANT = "Extract to constant";
 
     public static final String EXTRACT_TO_VARIABLE = "Extract to local variable";
+    
+    public static final String BALLERINA_RENAME_ACTION = "ballerina.action.rename";
```

Comment:
Is this the best place we can put this? @IMS94 @malinthar

Reply:
`ballerina.action.rename` is a command residing in the vscode plugin side. So, I'm ok with the place. Only issue is the name of the variable. Shall we name it `RENAME_COMMAND`?